feature/qos/tests/qos_policy_config_test/qos_policy_config_test.go

```diff
@@ -900,35 +977,31 @@ func testQoSCiscoClassifierConfig(t *testing.T) {
 			return out
 		})
 
-		if tc.classType == oc.Qos_Classifier_Type_IPV4 {
+		if tc.name == "dscp_based_classifier_ipv4" {
```

Comment:
Why are we comparing tc name instead of tc type? Looks to me it is not a good idea to compare strings.

Reply:
The original tests function which ConfigureQoS had similar  way to differentiate between v4 and v6 conditions was with comparing with string.	
condition := term.GetOrCreateConditions()
		if tc.name == "dscp_based_classifier_ipv4" {
			condition.GetOrCreateIpv4().SetDscpSet(tc.dscpSet)
		} else if tc.name == "dscp_based_classifier_ipv6" {
			condition.GetOrCreateIpv6().SetDscpSet(tc.dscpSet)
		}

But since in the previous releases , we did not support different policy-map name for V4 or v6 under same , I used type to modify the condition. Since in this release , we support the same, changed the test back how the original test was written.
Please let me know if we still have to change the test by using the type to derive the condition , I can do the same

Reply:
Fixed by using classType

Reply:
I am still seeing tc.name == "dscp_based_classifier_ipv4"..

Reply:
I missed the  code in get part. Fixed it now

Reply:
I am stilling seeing if tc.name == "dscp_based_classifier_ipv4", and I remember we agreed to tc.classType for comparison in the meeting?